h2o-core/src/test/java/water/parser/ChunksizeTest.java

```diff
@@ -103,14 +104,18 @@ public void run() throws IOException {
 
                 if (parseChunkCountPerNode*numCols > (1<<24)) {//no more than 16M chunk POJOs per node
                   msg += "MANY ";
-                  FileVec.calcOptimalChunkSize((long) totalSize, numCols, maxLineLength, cores, cloudSize, oldheuristic==1, true);
+                  FileVec.calcOptimalChunkSize((long) totalSize, numCols, maxLineLength, cores, cloudSize, oldheuristic==1, VERBOSE);
                   toomany[oldheuristic]++;
                   fail = true;
-                  Assert.assertTrue(totalSize/cloudSize/cores > 1e9); //only for big data, where we have more than 1GB per core
+
+                  final double actual = totalSize / cloudSize / cores;
+                  Assert.assertTrue(
+                      "Total " + totalSize + ", cloud size" + cloudSize +", cores " + cores + " => " + actual, 
+                      actual > 1e9); //only for big data, where we have more than 1GB per core
                 }
 
                 if (fail) {
-                  Log.info(msg + (oldheuristic==0?"(New Heuristic)":"(Old Heuristic)"));
+//                  Log.info(msg + (oldheuristic==0?"(New Heuristic)":"(Old Heuristic)"));
```

Comment:
Because it gives no information, but fills the output with tons of unnecessary lines.